config/webpack/commonWebpackConfig.js

```
@@ -1,35 +1,18 @@
-// The source code including full typescript support is available at:
-// https://github.com/shakacode/react_on_rails_tutorial_with_ssr_and_hmr_fast_refresh/blob/master/config/webpack/commonWebpackConfig.js
+// The source code including full typescript support is available at: 
```

ESLint

[error] Delete `·`

(prettier/prettier)